README.md

```diff
@@ -1,4 +1,6 @@
-**Earth Enterprise** is the open source release of Google Earth Enterprise, a geospatial application which provides the ability to build and host custom 3D globes and 2D maps.  Earth Enterprise does not provide a private version of Google imagery that's currently available in Google Maps or Earth.
+# Earth Enterprise [![Slack Status](https://img.shields.io/badge/chat-slack-ff69b4.svg)](http://slack.opengee.org) [![Join the chat at https://gitter.im/earthenterprise/Lobby](https://badges.gitter.im/earthenterprise/Lobby.svg)](https://gitter.im/earthenterprise/Lobby?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
```

Comment:
Just referring to `![Slack Status]`, which is the alt text format in markdown. Screen readers will read the link as Slack Status, but that's kind of a misnomer as the slack channel's status won't be displayed.